pkg/frontend/admin_openshiftcluster_etcdcertificaterenew.go

```diff
@@ -446,13 +446,13 @@ func (e *etcdrenew) isEtcdRevised(ctx context.Context) (bool, error) {
 	isAtRevision := true
 	rawEtcd, err := e.k.KubeGet(ctx, "etcd.operator.openshift.io", "", "cluster")
 	if err != nil {
-		e.log.Warnf(err.Error())
+		e.log.Warnf("%s", err.Error())
```

Comment:
Instead of doing the no-op formatting, prefer

```go
e.log.Warn(err.Error())
```

Reply:
Addressed